src/main/java/com/connexta/search/index/Index.java

```diff
@@ -25,12 +25,12 @@
 @NoArgsConstructor
 public class Index {
```

Comment:
❓ This class needs to be expanded to include all the new attributes, right?

Reply:
My PR adds the solr schema which will need to be modified to allow accepting of these attributes. I also introduced solr itests to test the solr schema that will need to be modified.

Reply:
Another great question. And one I hope to punt until the ticket in the next sprint that adds indexing for the new attributes.